Package.swift

```diff
@@ -27,6 +27,7 @@ let package = Package(
         .Package(url: "https://github.com/IBM-Swift/Kitura.git", majorVersion: 1, minor: 3),
         .Package(url: "https://github.com/IBM-Swift/HeliumLogger.git", majorVersion: 1, minor: 3),
         .Package(url: "https://github.com/IBM-Swift/Kitura-MustacheTemplateEngine.git", majorVersion: 1, minor: 1),
-        .Package(url: "https://github.com/IBM-Swift/Kitura-StencilTemplateEngine.git", majorVersion: 1, minor: 3),
+        .Package(url: "https://github.com/IBM-Swift/Kitura-StencilTemplateEngine.git", majorVersion: 1, minor: 4),
+        .Package(url: "https://github.com/kylef/Stencil", majorVersion: 0, minor: 7)
```

Comment:
In an ideal world, as long as module A's `Package.swift` follows SemVer, and modules B and C, as long as they follow SemVer when tagging releases, will not introduce a breaking change that would cause module A to break.

However, if you explicitly include module C in module A's `Package.swift`, even if a non-breaking update to module C was released, module A might break WITHOUT any change in module A's source code whatsoever.

This, to me, is a big no-no. We don't want, say, a tagged release of module A to suddenly break simply because of a dependency graph error introduced by an update to a dependency that both module A and module C rely on.

How could this happen?
Say module A is tagged 1.2.0, and relies on version 1.5.x of module B. Version 1.5.1 of module B depends on version 2.3.x of module C. Module C just got updated to version 2.4.0 from 2.3.7 (this IS NOT a breaking API change according to SemVer). Then, module B is updated to version 1.5.2 to pull in version 2.4.0 of module C.

Now, version 1.2.0 of module A is broken because of dependency graph error due to module C 2.3.7->2.4.0 update. This means that all projects that rely on version 1.2.0 of module A is now suddenly broken even though there has been no change to module A whatsoever, and that the module C update is a non-breaking API change according to SemVer.

TL;DR: Having redundant packages listed in `Package.swift` can break an existing, tagged release through no fault of its own.